src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java

```diff
@@ -900,4 +912,17 @@ public static void enterKeyValuePairsForProperty(String pluginProperty, String j
       index++;
     }
   }
+  /**
+   * Enter Credentials/Authorization values in the Plugin Property (input)
+   *
+   * @param pluginProperty @data-cy attribute value of Plugin Property.
+   *                       If pluginProperty is present in {@link ConstantsUtil#DEFAULT_DATACY_ATTRIBUTES_FILE}
+   *                       then its data-cy is fetched from it
+   *                       else pluginProperty is used as it is.
+   * @param value          If the value is present in the Environment variables then its value is fetched
+   */
+  public static void enterCredentialsInInputProperty(String pluginProperty, String value) {
```

Comment:
How is this method different from old implementation of `enterValueInInputProperty()`?

Reply:
I see, this method is directly populating the input field with the value passed in param. Why do we need to explicitly specify `Credentials` in method name?
Also I don't see the operation of fetching the property from environment variable being performed in this method according to the description.

Reply:
I would rather like to rename the method to `enterValueInInputPropertyFromEnv` and do `System.getenv(PluginPropertyUtils.pluginProp(envVariableKey)` inside this method rather than in the caller.